resources/views/pages/products.ejs

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <% include ../partials/head %>
+    <link rel="stylesheet" href="/resources/css/products.css" />
```

Comment:
Create another CSS file which should have CSS common across pages or of atomic components extracted in a separate file.

Reply:
Yes, I have created _common.scss and it is imported in these pages.